Libplanet.Extensions.Cocona.Tests/Commands/StoreCommandTest.cs

```diff
@@ -2,23 +2,36 @@
 using System.Collections.Immutable;
 using System.IO;
 using Cocona;
+using Libplanet.Blocks;
+using Libplanet.Crypto;
 using Libplanet.Extensions.Cocona.Commands;
 using Libplanet.RocksDBStore.Tests;
+using Libplanet.Tests;
 using Libplanet.Tests.Store;
+using Libplanet.Tx;
 using Xunit;
 
 namespace Libplanet.Extensions.Cocona.Tests.Commands
 {
     public class StoreCommandTest : IDisposable
     {
         private readonly ImmutableArray<StoreFixture> _storeFixtures;
-        private readonly StoreCommand _command;
         private readonly TextWriter _originalWriter;
+        private readonly Block<Utils.DummyAction> _genesisBlock;
+        private readonly Block<Utils.DummyAction> _block1;
+        private readonly Block<Utils.DummyAction> _block2;
+        private readonly Block<Utils.DummyAction> _block3;
+        private readonly Block<Utils.DummyAction> _block4;
+        private readonly Block<Utils.DummyAction> _block5;
+        private readonly Transaction<Utils.DummyAction> _transaction1;
+        private readonly Transaction<Utils.DummyAction> _transaction2;
+        private readonly Transaction<Utils.DummyAction> _transaction3;
+        private readonly Transaction<Utils.DummyAction> _transaction4;
```

Comment:
How about making an array?

Reply:
I am not sure about the benefit of doing that.
Is there any weak point declaring 4 individual variables in this test case? @limebell

Reply:
Just because it's clean? Not a strong suggestion though. Please check

Reply:
IMHO, those variables don't share any semantics, such as living in the same block or signed by the same signer, in the following testcases. So I think that there is no need to aggregate them. And the beauty or cleanliness of code is up to the individuals unless it does not break the coding rule(lint)

Hence If you don't mind, may I keep them as a separate variable?

Reply:
🆗